apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.Tests/FakeStorageService.cs

```diff
@@ -44,33 +36,18 @@ class FakeStorageService : StorageService
 
         public void ExpectRequest<TResponse>(ClientServiceRequest<TResponse> request, TResponse response)
         {
-            //MaybeDisablePrettyPrint(request);
             var httpRequest = request.CreateRequest();
             string responseContent = SerializeObject(response);
             handler.ExpectRequest(httpRequest.RequestUri, httpRequest.Content?.ReadAsStringAsync()?.Result, responseContent);
         }
 
         public void ExpectRequest<TResponse>(ClientServiceRequest<TResponse> request, HttpStatusCode statusCode)
         {
-            // TODO: Validate that this isn't a success status code (given that we're returning an error response).
-
-            //MaybeDisablePrettyPrint(request);
             var httpRequest = request.CreateRequest();
-            var responseMessage = new HttpResponseMessage(statusCode)
-            {
-                Content = new StringContent(SampleErrorContent)
-            };
+            var responseMessage = new HttpResponseMessage(statusCode);
```

Comment:
This now appears to be back to the original "not having any content at all" - which made it hard to understand the error. It's okay while the tests are passing, but it will be confusing on some failures. I'd create a method to return an appropriate StringContent for the given status code - but that could come after this PR.

Reply:
Sure we can put it in the next PR